src/apps/appstore/app.js

```diff
@@ -534,7 +534,13 @@ define(function(require) {
 			const self = this;
 			const configForm = template.find('#configuration_form');
 
-			monster.ui.validate(configForm);
+			monster.ui.validate(configForm, {
+				rules: {
+					api_url: {
+						url: true
```

Comment:
This checks is rejecting using private IPs like 192.168.0.1 and etc. For now please remove this, since it makes the form invalid and I can't update anything because of this.